sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -1070,7 +1070,6 @@ public class ServiceBusMultipleMessagesTestJob_BindToMessageArray
                 DateTimeOffset[] expiresAtArray,
                 DateTime[] enqueuedTimeUtcArray,
                 DateTimeOffset[] enqueuedTimeArray,
-                DateTimeOffset[] lockedUntilArray,
```

Comment:
This was removed in a later commit of https://github.com/Azure/azure-sdk-for-net/pull/27308